bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -11,6 +11,7 @@
 
 		<properties>
 			<property name="vendor">Select Comfort</property>
+			<property name="thingTypeVersion">1</property>
```

Comment:
This should be in the updated thing, not in the bridge.

Reply:
Yes, of course. Fixed.